dom-element.js

```diff
@@ -93,25 +93,43 @@ DOMElement.prototype.insertBefore =
         return elem
     }
 
-DOMElement.prototype.setAttribute =
-    function _Element_setAttribute(name, value) {
-        this._attributes[name] = value
+DOMElement.prototype.setAttributeNS =
+    function _Element_setAttributeNS(namespace, name, value) {
+        var localName = name.substr(name.indexOf(":") + 1)
```

Comment:
@Raynos Yes, the part before `:` is the namespace. The spec is kind of weird because [`setAttributeNS`](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-ElSetAttrNS) expects a namespace-qualified name, but [`getAttributeNS`](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-ElGetAttrNS) and [`removeAttributeNS`](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-ElRemAtNS) expect a local name (without the namespace prefix).